### PR TITLE
[DOC] Add namespace doc for Java part.

### PR DESCRIPTION
**Pull request by jovany-wang**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by jovany-wang** [comment]
@wuisawesome CC that I changed a little python part.

**Comment by jovany-wang** [comment]
Since we do a lot of Java namespace, this should be checked in 1.9 so that it can be referred.

File: doc/source/namespaces.rst [hunk 1]
```diff
@@ -48,6 +56,55 @@ Named actors are only accessible within their namespaces.
...
+    }
+
+    // Job 3 connects to the original "colors" namespace.
+    context = ray.init("ray://localhost:10001", namespace="colors")
+    # This fails because "watermelon" was in the fruits namespace.
+    ray.get_actor("watermelon")
+    # This returns the "orange" actor we created in the first job, not the second.
+    ray.get_actor("orange")
+    context.disconnect()
+    # We are manually managing the scope of the connection in this example.
```
Comment (kfstorm) [inline doc/source/namespaces.rst:95]: Copied the Python code and forgot to update it to Java?

File: doc/source/namespaces.rst [hunk 2]
```diff
@@ -48,6 +56,55 @@ Named actors are only accessible within their namespaces.
     context.disconnect()
     # We are manually managing the scope of the connection in this example.
 
+  .. code-tab:: java
+
+    // `ray start --head` has been run to launch a local cluster.
+
+    // Job 1 creates two actors, "orange" and "purple" in the "colors" namespace.
+    System.setProperty("ray.address", "127.0.0.1:6379");
```
[the remark this answers is not in the document]
Comment (jovany-wang) [reply doc/source/namespaces.rst:64]: I don't make sure if it works within `ray://` protocol label in Java worker.

File: doc/source/namespaces.rst [hunk 3]
```diff
@@ -9,17 +9,25 @@ named, its name must be unique within the namespace.
...
-   ray.init("ray://<head_node_host>:10001", namespace="world")
+    ray.init(namespace="hello")
+    # or using ray client
+    ray.init("ray://<head_node_host>:10001", namespace="world")
+
+  .. code-tab:: java
+
+    System.setProperty("ray.job.namespace", "hello"); // set it before Ray.init()
+    Ray.init();
+    // Or you can specified the namespace in your configure file.
```
[the remark this answers is not in the document]
Comment (jovany-wang) [reply doc/source/namespaces.rst:23]: Hum, it seems we miss many options yet.

File: doc/source/namespaces.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (kfstorm) [reply doc/source/namespaces.rst:66]: I don't think we support `ray://` prefix in Java. But we should be able to use `localhost:10001`.

[the remark this answers is not in the document]
Comment (jovany-wang) [reply doc/source/namespaces.rst:69]: Hum, I see. Great catch.

File: doc/source/namespaces.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jovany-wang) [reply doc/source/namespaces.rst]: Added it to driver options section.

**Comment by jovany-wang** [comment]
@kfstorm Comments were addressed.

**Review by kfstorm — APPROVED** [review]

**Comment by jovany-wang** [comment]
@iycheng Could you pick this commit to the releasing branch?

**Pull request merged**